docs/data-sources/as_policy_execute_logs.md

```diff
@@ -0,0 +1,114 @@
+  If omitted, the provider-level region will be used.
+
+* `scaling_policy_id` - (Required, String) Specifies the scaling policy ID.
+
+* `log_id` - (Optional, String) Specifies the policy execution log ID.
+
+* `scaling_resource_id` - (Optional, String) Specifies the scaling resource ID.
+
+* `scaling_resource_type` - (Optional, String) Specifies the scaling resource type.
+  The value can be **SCALING_GROUP** or **BANDWIDTH**.
+
+* `execute_type` - (Optional, String) Specifies the policy execution type.
+  The value can be **SCHEDULED**, **RECURRENCE**, **ALARM** or **MANUAL**.
+
+* `start_time` - (Optional, String) Specifies the start time of the policy execution used for query. The time format is
+  **yyyy-MM-ddThh:mm:ssZ**.  
+  The query result is all data with a policy execution time greater than or equal to this value.
+
+* `end_time` - (Optional, String) Specifies the end time of the policy execution used for query. The time format is
+  **yyyy-MM-ddThh:mm:ssZ**.  
+  The query result shows all data with policy execution time less than this value.
+
+* `status` - (Optional, String) Specifies the policy execution status. The value can be **SUCCESS**, **FAIL**
+  or **EXECUTING**.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The data source ID.
+
+* `execute_logs` - All policy execution logs that match the filter parameters.  
+  The [execute_logs](#as_execute_logs_attr) structure is documented below.
+
+<a name="as_execute_logs_attr"></a>
+The `execute_logs` block supports:
+
+* `id` - The policy execution log ID.
+
+* `status` - The policy execution status.
+
+* `failed_reason` - The reason of policy execution failure.
+
+* `execute_type` - The policy execution type.
+
+* `execute_time` - The policy execution time, the time format is **yyyy-MM-ddThh:mm:ssZ**.
+
+* `scaling_policy_id` - The scaling policy ID.
+
+* `scaling_resource_id` - The scaling resource ID.
+
+* `scaling_resource_type` - The scaling resource type.
+
+* `old_value` - The scaling original value.
+  + When `scaling_resource_type` is **SCALING_GROUP**, this field represents the number of instances.
+  + When `scaling_resource_type` is **BANDWIDTH**, this field represents the bandwidth size, in Mbit/s.
+
+* `desire_value` - The scaling target value.
+  + When `scaling_resource_type` is **SCALING_GROUP**, this field represents the number of instances.
+  + When `scaling_resource_type` is **BANDWIDTH**, this field represents the bandwidth size, in Mbit/s.
+
+* `limit_value` - The operational limitations. When `scaling_resource_type` is **BANDWIDTH** and `type` is not **SET**,
+  this field takes effect in Mbit/s.
+  + When `type` is **ADD**, this field represents the maximum bandwidth limit.
+  + When `type` is **REMOVE**, this field represents the minimum bandwidth limit.
+
+* `type` - The policy execution task type. The value can be **REMOVE**, **ADD** or **SET**.
```

Comment:
I suggest you to move this attribute before the `old_value`, so that customs can understand this attribute first and will not confuse why is the value can be set with the **SET** according to the `limit_value` description.

Reply:
fixed.